app/policies/workspace_policy.rb

```diff
@@ -11,6 +11,44 @@ def create_invitation?
     user.admin?(workspace)
   end
 
+  def add_member?(channel)
+    return false if channel.name == 'general'
```

Comment:
I see this `general` channel is being used in many places, and you are adding essential logic based on it. What if you move this name to a constant in the channel model so it's easy to change later if needed?

Reply:
ok, done, what do you think?

[...]
```ruby
BASIC_CHANNEL_NAME = 'general'.freeze
```
[...]